xarray/core/computation.py

```diff
@@ -1338,3 +1339,69 @@ def polyval(coord, coeffs, degree_dim="degree"):
         coords={coord.name: coord, degree_dim: np.arange(deg_coord.max() + 1)[::-1]},
     )
     return (lhs * coeffs).sum(degree_dim)
+
+
+def _calc_idxminmax(
+    *,
+    array,
+    func: Callable,
+    dim: Optional[Hashable],
+    skipna: bool = None,
+    fill_value: Any = None,
```

Comment:
Should this be `dtypes.NA`? Or maybe we don't need default values at all, given that this is only called internally?

Reply:
I'll fix that.